findbugs/build.gradle

```diff
@@ -178,7 +178,7 @@ task unzipDist(type:Copy, dependsOn:distZip) {
 task smokeTest {
   dependsOn unzipDist
   doLast {
-    ant.taskdef(name:'spotbugs', classname:'edu.umd.cs.findbugs.anttask.FindBugsTask', classpath:jar.outputs.files.asPath)
+    ant.taskdef(name:'spotbugs', classname:'edu.umd.cs.findbugs.anttask.FindBugsTask', classpath:project(':spotbugs-ant').jar.outputs.files.asPath)
```

Comment:
if you are to do this, you should add `project(':spotbugs-ant')` to this task's `dependsOn` at line 179.